sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml

```diff
@@ -6,6 +9,7 @@ azure:
     user-group:
       allowed-group-names: group1,group2
       allowed-group-ids: <group1-id>,<group2-id>
+      enable-full-list: false
     post-logout-redirect-uri: http://localhost:8080
```

Comment:
I think we should write some description about the param `enable-full-list` in readme.md

Reply:
get it.